### ISSUE TITLE
Installing using version => 'stable' installs v5.0.0 instead of v5.1.1

### ISSUE THREAD

**Issue by mikijov** [willdurand/puppet-nodejs#107]
When installing node using `version => 'stable'`, version v5.0.0 is picked up, even though v5.1.1 is the latest stable build. BTW, when using `version => 'v5.1.1'`, then v5.1.1 is properly installed.

**Comment by Ma27** [comment]
seems to be a bug, I'll try to reproduce in the next time

**Comment by Ma27** [comment]
I can't reproduce this anymore. When I write

``` puppet
class { '::nodejs':
  version => 'stable',
  make_install => false,
}
```

and use a trusty64 environment then Node v5.4.1 (it's currently the latest one) will be installed.

**Comment by Ma27** [comment]
might have been an issue in the old installer, fixed in `2.0.0-alpha1`
